### DISCUSSION TITLE
Sign PSBT on openordex using ord wallet

### DISCUSSION THREAD

**Discussion by goddesslens** [ordinals/ord#2003] [General]
How to sign PSBT on openordex using ord wallet on windows ?

**Comment by cliobitcoinbank** [comment]
You can do it with Sparrow Wallet and there is a video guide on how to use sparrow to do it posted here: https://twitter.com/orenyomtov/status/1628938029682548737
You can read more on the Bitcoin Ordinals reddit wiki here: https://www.reddit.com/r/BitcoinOrdinals/wiki/index/resources/ It may be a more public place to ask questions and get community support.
